simulation_ws/src/object_tracker_simulation/package.xml

```diff
@@ -9,16 +9,22 @@
   <author email="ros-contributions@amazon.com">AWS RoboMaker</author>
   <maintainer email="ros-contributions@amazon.com">AWS RoboMaker</maintainer>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament</buildtool_depend>
 
+  <depend>rclpy</depend>
   <depend>gazebo_ros</depend>
   <depend>gazebo_plugins</depend>
   <depend>geometry_msgs</depend>
   <depend>nav_msgs</depend>
-  <depend>roscpp</depend>
   <depend>sensor_msgs</depend>
   <depend>std_msgs</depend>
-  <depend>tf</depend>
 
   <exec_depend>gazebo</exec_depend>
+  <exec_depend>turtlebot3_bringup</exec_depend>
```

Comment:
Why do we need to depend on bringup for simulation? and how come we don't need other packages like turtlebot3_description, turtlebot3_simulations, turtlebot3_gazebo?

Reply:
> Why do we need to depend on bringup for simulation?

Agree. Here turtlebot3_bringup turtlebot3 simulation dependency is not required, Hence we have removed it.


> how come we don't need other packages like turtlebot3_description, turtlebot3_simulations, turtlebot3_gazebo?

We have created equivalent package of "turtlebot3_simulation_description" for "turtlebot3_description_reduced_mesh". will commit it in next PR. 
All turtlebot3 simulation dependencies such as turtlebot3_description, turtlebot3_simulations, turtlebot3_gazebo etc. are required and included in "turtlebot3_simulation_description" as turtlebot3 and it gets installed in src/deps using .rosinstall.